inst/@sym/assumeAlso.m

```diff
@@ -64,53 +64,56 @@
 %% @seealso{@@sym/assume, assumptions, sym, syms}
 %% @end deftypemethod
 
-%% Author: Colin B. Macdonald
-%% Keywords: symbolic
 
 function varargout = assumeAlso(x, varargin)
 
   assert (nargin > 1, 'General algebraic assumptions are not supported');
 
-  [tilde,ca] = assumptions(x, 'dict');
+  for i = 1:length (x)
+
+    xstr = subsref(x, substruct('()', {i}));
```

Comment:
mm, taht will change all the other syntax.., or add (at least to me) useless lines, because finally this will be:
```
x = ...
[tilde, ca] ...
xstr = x
```
or if i replace all xstr to x i will need change the names of the vars in assume and assumeAlso, i think you prefer the last option? (or keep how is now)